### PR TITLE
Update install instructions to match COMPAS

### PR DESCRIPTION
**Pull request by gonzalocasas**
Main reason for this PR was the line added in `conf.py` to get rid of these annoying artifacts that appeared in the docs a while ago:
image

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [x] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.rst` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_fab.robots.Configuration`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -4,45 +4,82 @@ Getting started
...
-But it can also be installed using ``pip``:
+.. code-block:: bash
 
-::
+    conda activate my-project
+    python -c "import compas_fab; print(compas_fab.__version__)"
+
+.. code-block:: none
+
+    0.17.0
```
Comment (beverlylytle) [inline docs/getting_started.rst:31]: Should there be a hook to update this when the version is bumped?

Comment (beverlylytle) [reply docs/getting_started.rst:31]: Oh, sorry, missed it.

Comment (gonzalocasas) [reply docs/getting_started.rst:31]: 😉

Comment (beverlylytle) [reply docs/getting_started.rst:31]: (this hook doesn't exist in compas)

**Review by beverlylytle — APPROVED** [review]
LGTM!

**Pull request merged**